### ISSUE TITLE
Block mail domain seznam.cz

### ISSUE THREAD

**Issue by erickskrauch** [elyby/accounts#9]
We received a lot of false-positive abuse reports for this mail service. To protect reputation of our domain I want to disallow to interact with this mailing service.

**Comment by the-nexusmc** [comment]
here in 2025 , why for saint hedwig of silesia did you stop interacting with seznam.cz ? i just wanna make a damn account

**Comment by erickskrauch** [comment]
Exactly because of the reasons, explained in the issue.

**Comment by the-nexusmc** [comment]
yeah i know but that kind of throws a brick into making a ely.by account
